plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -307,6 +317,63 @@ public void testMissingColumnsOnInsert()
         assertQuery("SELECT * FROM test_col_insert", "SELECT 1, 'val1', 'val2'");
     }
 
+    @Override
+    public void testTopNPushdown()
```

Comment:
let's give it a better name

otherwise it may escape reader's attention that TopN is not pushed down, but partial TopN is replaced with partial Limit

Reply:
We would have to change it in BaseJdbcConnectorTest and all classes overriding this method.
Perhaps better in a separate commit...?

Reply:
I also need to override testTopNPushdown since otherwise it would fail. I could also override testTopNPushdown here to do nothing, and then add another test that verifies the plan modification for partial Limit.

Or rename to testTopNBehavior?

Reply:
I think you can override this test to do nothing (with comment why) and move the tests to `testReplacePartialTopNWithLimit`

Reply:
Did that.